uaclient/apt.py

```diff
@@ -611,8 +610,6 @@ def get_apt_cache_time() -> Optional[float]:
     cache_time = None
     if os.path.exists(APT_UPDATE_SUCCESS_STAMP_PATH):
         cache_time = os.stat(APT_UPDATE_SUCCESS_STAMP_PATH).st_mtime
-    elif os.path.exists(APT_LISTS_PATH):
-        cache_time = os.stat(APT_LISTS_PATH).st_mtime
```

Comment:
@renanrodrigo I had to make these changes because the apt list path gets updated before the apt-news script runs. So if for any reason we use it's timestamp, then apt-news will think that apt update was called <24h ago and end early.

LMK if that makes sense and if you think this change is okay.

Reply:
I think this change is fine.
Lists were used here only because that's what ansible does (I stole the code from there), but ansible needs more general purposes things.
The stamp file should be enough because it is present in every Ubuntu > 16.04, which is what we need anyway.